skycoin/skywire
# Add the persistent transports to the UI

Did you run `make format && make check`?
The go code was not changed. `npm run lint` and `npm run build` were used.

Fixes #862

 Changes:	
- Now the transport list does not have the online/offline indicator.
- Now the transport list has a column for checking if a transport is persistent or not. The icon can be used for setting the transport as persistent or non-persistent.
- Options for making persistent/non-persistent all the selected transports was added.

How to test this PR:
Open the visor details page in the Skywire manager and use the options. You can make the transport persistent/non-persistent by clicking the star icon in the transport row. You can make several transports persistent/non-persistent by selecting the checkbox at the left of the transport row, then clicking the 3 dots button at the top right part of the list and then selecting the option you want to use.

NOTE: @ersonp after checking how the API works, it appears that at the end having an endpoint just for getting the persistent transports list would be good. It is not totally needed, as the data can be obtained from the summary endpoint, but it would allow to improve the performance.
